code/ecdsap256/Hacl.P256.fsti

```diff
@@ -258,16 +311,42 @@ val verify_q:
       )
     )
 
+[@@ Comment "Validate a private key.
+
+Input: scalar: uint8[32].
+  \n Output: bool, where true stands for the scalar to be more than 0 and less than order."]
+val validate_private_key: x: lbuffer uint8 (size 32) -> Stack bool
```

Comment:
Any reason why you need a private key validation?

Reply:
I think it could be needed only for ecdsa

Reply:
it was in the API to begin with, just with a surprising name; it is also exposed as a private key validation function by the OCaml bindings (see https://hacl-star.github.io/ocaml_doc/hacl-star/Hacl_star/Hacl/P256/index.html)